pyproject.toml

```diff
@@ -1,18 +1,19 @@
-[tool.poetry]
-name = "django-modeltranslation"
-version = "0.14.4"
-description = "Translates Django models using a registration approach."
-authors = ["Peter Eschler <peschler@gmail.com>"]
-maintainers = ["Serhii Tereshchenko <serg.partizan@gmail.com>"]
-
-[tool.poetry.dependencies]
-python = "^3.8"
-django = "^3.0.4"
-
-[tool.poetry.dev-dependencies]
-pdbpp = "^0.10.2"
-flake8 = "^3.7.9"
-
-[build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+[tool.poetry]
+name = "django-modeltranslation"
+version = "0.14.4"
+description = "Translates Django models using a registration approach."
+authors = ["Peter Eschler <peschler@gmail.com>"]
+maintainers = ["Serhii Tereshchenko <serg.partizan@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+django = "^3.0.4"
+
+[tool.poetry.dev-dependencies]
+pdbpp = "^0.10.2"
+flake8 = "^3.7.9"
+six = "^1.15.0"

```

Comment:
Why are your files contains ^M characters?

https://stackoverflow.com/questions/1889559/git-diff-to-ignore-m

Reply:
I have no idea... Maybe because I'm on Windows?
Maybe if I just edit the file on Github they stop showing?

Reply:
Should I delete this file? 
I've only added six because the test suite wouldn't run otherwise, but it not integral to the PR.